tests/resource_tests/generator_tests/test_workbench.py

```diff
@@ -82,3 +189,15 @@ def test_workbench_researcher_generator(self):
         self.assertEqual(data['given_name'], 'test')
         self.assertEqual(data['genders'], [{'gender': 'MAN', 'gender_id': 1}, {'gender': 'WOMAN', 'gender_id': 2}])
         self.assertEqual(data['degrees'], [{'degree': 'PHD', 'degree_id': 1}, {'degree': 'MBA', 'degree_id': 8}])
+
+    def test_workbench_researcher_dsv2_generator(self):
+        self.session.add(self.researcher2)
+        self.session.add(self.researcher3)
+        self.session.commit()
+
+        gen = WBResearcherGenerator()
+        res = gen.make_resource(2)
+        self.assertIsNotNone(res)
```

Comment:
Besides `assertIsNotNone(res)`, are there any other finer-grained assertions to validate on the PDR generator record?

Reply:
The test was just to make sure that both a pre-DSV2 and post-DSV2 record won't throw an exception in the generator.